packages/evolution-frontend/src/components/inputs/InputChoiceSorting.ts

```diff
@@ -198,7 +196,7 @@ const customHorizontal = function<Type>(arr: Type[], custom: number[]): Type[][]
    * @param customAlignmentLengths - Specify the lengths of each row or column individually, if alignement is horizontal, the first specified row must be the longuest.
    * @returns A 2 dimension array containing the input array sorted into columns for display or the original array in case of invalid parameters.
    */
-const sortByParameters = function<Type>(arr: Type[], alignment?: 'vertical' | 'horizontal' | 'auto', columns?: number, rows?: number, customAlignmentLengths?: number[]): Type[] | Type[][] {
+const sortByParameters = function<T>(arr: T[], alignment?: 'vertical' | 'horizontal' | 'auto', columns?: number, rows?: number, customAlignmentLengths?: number[]): T[] | T[][] {
```

Comment:
I think it is now impossible to have simply `T[]` as return type. The function now has a `else` with a `switch` that will always return one of the above method, so the return type should just be `T[][]`